content/docs/2.0/scalers/influxdb.md

```diff
@@ -0,0 +1,61 @@
++++
```

Comment:
Nit - The official name is `InfluxDB`, can we rename all `Influxdb` to this please?